pkg/skaffold/build/scheduler.go

```diff
@@ -90,11 +91,16 @@ func (s *scheduler) build(ctx context.Context, tags tag.ImageTags, i int) error
 
 	event.BuildInProgress(a.ImageName)
 	eventV2.BuildInProgress(i, a.ImageName)
+	ctx, endTrace := instrumentation.StartTrace(ctx, "build_BuildInProgress", map[string]string{
+		"ImageName": instrumentation.PII(a.ImageName),
+	})
```

Comment:
Can we incorporate the builder type?  And the artifact number `i`?

Reply:
Is the builder type accessible in `scheduler.go` here?  Current approach would be to do something like adding attributes to the current trace (via otel [SpanFromContext](https://github.com/opentracing/opentracing-go/blob/3088eee7e4d26010ac9a301e977fd2721dbebcbe/gocontext.go#L20-L32))

Reply:
Done